doc/source/development/contributing.rst

```diff
@@ -204,6 +204,8 @@ You will need `Build Tools for Visual Studio 2017
 	You DO NOT need to install Visual Studio 2019.
 	You only need "Build Tools for Visual Studio 2019" found by
 	scrolling down to "All downloads" -> "Tools for Visual Studio 2019".
+        
+	In the Visual Studio Installer, install C++ build tools and ensure the latest versions of MSVCv142 - VS 2019 C++ x64/x86 build tools and Windows 10 SDK are checked.
```

Comment:
MIght want to line wrap around 80 which is an approximate standard

Reply:
I'm not sure these instructions are correct.  From my own installation, which works correctly, I only have:

![image](https://user-images.githubusercontent.com/5585221/90113403-346aff80-dd49-11ea-92b9-329863a2b9cd.png)

v140 is the key dependency and corresponds to VS2015, which is the compiler for the Python 3.5+ on Windows.

Reply:
Thanks @bashtage, I'll fix up the line length. 

I got the impression from reading https://wiki.python.org/moin/WindowsCompilers that the latest 14.x version of Visual C++ is recommended. I couldn't find any other page that talks about the versions. Are the components (and versions) you have installed preferable to ensure consistency amongst pandas developers?